src/languages/sql/sql.formatter.ts

```diff
@@ -70,7 +70,7 @@ export default class SqlFormatter extends Formatter {
       reservedDependentClauses: ['WHEN', 'ELSE'],
       reservedKeywords: keywords,
       reservedFunctionNames: functions,
-      stringTypes: [{ quote: "''", prefixes: ['X'] }],
+      stringTypes: [{ quote: "''", prefixes: ['X', 'N'] }],
```

Comment:
Looks like the standard also supports `U&''` strings: https://jakewheat.github.io/sql-overview/sql-2008-foundation-grammar.html#Unicode-character-string-literal

Should also document these all in the wiki.